spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1442,6 +1436,33 @@ def check_value(name, value):
     qtbot.keyClicks(control, '!a.ba')
     qtbot.keyClick(control, Qt.Key_Tab)
     qtbot.waitUntil(lambda: control.toPlainText().split()[-1] == '!a.baba')
+    qtbot.keyClick(control, Qt.Key_Enter)
+    qtbot.waitUntil(lambda: control.toPlainText().split()[-1] == 'ipdb>')
+
+    # Check we can complete pdb command names
+    qtbot.keyClicks(control, 'longl')
+    qtbot.keyClick(control, Qt.Key_Tab)
+    qtbot.waitUntil(lambda: control.toPlainText().split()[-1] == 'longlist')
+
+    qtbot.keyClick(control, Qt.Key_Enter)
+    qtbot.waitUntil(lambda: control.toPlainText().split()[-1] == 'ipdb>')
+
+    # Check we can use custom complete for pdb
+    fd, path = tempfile.mkstemp('test.py')
```

Comment:
Please the `tmpdir` fixture instead of the `tempfile` module to deal with temporary files.